api-client/src/main/java/com/xing/api/CallSpec.java

```diff
@@ -172,6 +172,13 @@
      */
     CallSpec<RT, ET> formField(String name, List<String> values);
 
+    /**
+     * Overrides the connection's default timeouts for this particular call.
+     *
+     * All timeouts in seconds.
```

Comment:
Also it would be helpful to explain further what connectTimeout and readTimeout will actually do etc. using @param